### ISSUE TITLE
RStudio 1.3 hangs with "lt_LT_new" dictionary (Qt QtWebEngineProcess)

### ISSUE THREAD

**Issue by GegznaV** [rstudio/rstudio#6594]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.938-2 Preview (2020-04-07)
    OS Version      : Windows 10
    R Version       : 3.6.3

### Steps to reproduce the problem

1) I had RStudio 1.2.5033 installed.
2) I updated RStudio to 1.3.938.
3) I went to Tools → Global Options → Spelling
4) I downloaded additional dictionaries.
5) I chose the dictionary "lt_LT_new", enabled real-time spellchecking (see Fig. 1) and pressed "OK" to exit the Global Options pane. 
6) RStudio hanged. I had to use Task Manager to kill RStudio. But after RStudio was restarted, the problem still persisted.

**Update:** RStudio hangs even if real-time spellchecking is not enabled.
![image](https://user-images.githubusercontent.com/12725868/79246804-57ac9d80-7e82-11ea-95c6-2788844052f3.png)
**Fig. 1.** Modified options.

### Describe the problem in detail
With these settings, RStudio opens and panes are loaded but the program is frozen. I can do nothing, it is even impossible to write to the console. The Windows _Task Manager_ shows that RStudio uses a lot of computer resources (see the list of processes below). The situation did not change in 30 minutes when RStudio was opened. 
It seems that the problem may be related to the dictionary "lt_LT_new" and "Qt Qtwebengineprocess":

![image](https://user-images.githubusercontent.com/12725868/78423860-fc083600-7671-11ea-837a-f006341d444c.png)
**Fig. 2.** Processes in Task Manager.

**Notes:** 

1) When I roll back to 1.2.5033, RStudio works normally again. "Qt Qtwebengineprocess" takes only 141.4MB and power usage is "Very low".
2) If I install RStudio 1.3 again, the problem persists until I change the settings in `%APPDATA%\Roaming\RStudio\rstudio-prefs.json` from:
```json
    "spelling_dictionary_language": "lt_LT_new",
```
to:
```json
    "spelling_dictionary_language": "lt_LT",
```
and restart RStudio. `lt_LT` does not support real-time spell checking. But some other dictionaries (English US, German) do not cause the issue with real-time spellchecking enabled.

### Describe the behavior you expected

I expect real-time spellchecking works well with Lithuanian language and RStudio does not hang when this functionality is used.

<details>
 <summary>Issue checklist</summary>

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

</details>

**Comment by ronblum** [comment]
@GegznaV Thank you for filing the issue. I'll mark it for triage, to review as we continue development.

**Comment by kevinushey** [comment]
Does switching to software rendering make a difference? https://support.rstudio.com/hc/en-us/articles/360017886674-Troubleshooting-RStudio-Rendering-Errors

**Comment by GegznaV** [comment]
I read the link by @kevinushey and came up with an idea to delete some files with settings... And I managed to isolate the problem: it is related to the dictionary "lt_LT_new". I updated my original post and the steps on how to reproduce the issue. Could you reproduce it now?

@ronblum, I think the labels should be updated.

**Comment by kevinushey** [comment]
@adamconroy, does this sound familiar? Do we need to disable / disallow this dictionary?

**Comment by adamconroy** [comment]
@kevinushey yes, I might have missed this `lt_LT_new` dictionary in my testing. I actually don't think that dictionary should have been added to the zip file as I believe it was experimental. 

@GegznaV apologies but we don't currently support realtime spellchecking with Lithuanian due to the algorithmic complexity of its ruleset with the library that we are using. Manual spellchecking still works. This will be fixed in a future release. Please use the regular `lt_LT` dictionary and note that it disables the spellchecking. If you're stuck in a bad state, open your rstudio settings file and delete the spellcheck dictionary block.

**Comment by jmcphers** [comment]
Fixed in #6671.
